### ISSUE TITLE
minor issue with the definition

### ISSUE THREAD

**Issue by RobertHarper** [SMLFamily/Successor-ML#6]
The Definition as written cannot make up its mind whether it is working with abstract or concrete syntax.  The distinction between "atomic patterns" and "patterns" is a case in point.  I would say that the statics and dynamics is defined on abstract syntax, and eliminate all such holdovers from concrete syntax.

**Comment by JohnReppy** [comment]
I agree.  One of the things on my TODO list is a unified (and modern) description of the concrete syntax of Successor ML.  I think that we can then simplify the "syntax of the core" etc. to just refer to a normalized abstract syntax.
